molo/core/migrations/0078_add_translation_fields.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-09-03 12:40
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0003_auto_20180831_1312'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='articlepage',
+            options={'ordering': ('-latest_revision_created_at',), 'verbose_name': 'Article'},
+        ),
```

Comment:
What is the above migration for? Please find out where that change comes from. If this is a migration that someone else forgot to add to their work please make a separate migration and PR for it.

Reply:
I must have missed it from a pull. Thank you